src/activities/content/ContentModuleEntity.js

```diff
@@ -11,28 +12,22 @@ export class ContentModuleEntity extends Entity {
 	 * @returns {string} Description html of the content-module item
 	 */
 	descriptionRichText() {
-		if (!this._entity) {
-			return null;
-		}
-		const subEntity = this._entity.getSubEntitiesByClass(Classes.content.description)[0];
-		if (!subEntity || !subEntity.properties) {
+		const descriptionSubEntity = ContentHelperFunctions.getDescriptionSubEntity(this._entity);
+		if (!descriptionSubEntity) {
 			return null;
```

Comment:
this function says `* @returns {string} Description html of the content-module item`
so either we need to return an empty string or update that ^

Reply:
Yeah, it's your call. I handle the null case in the UI I believe so returning an empty string isn't necessary and would hide errors like you mentioned somewhere else. So updating the comment would probably be preferred.